src/style.css

```
@@ -0,0 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.blocs {
+  display: flex;
+  font-family: Arial, sans-serif;
+  color: #fff;
+  font-size: 100px;
+  flex-wrap: wrap;
+
+}
+
+.bloc {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 300px;
+  min-width: 33%;
+  flex-grow: 1;
+  flex-shrink: 0;
+}
+
+.color-1 {
+  background-color: #f00;
+  order: 1;
+}
+
+.color-2 {
+  background-color: #cd0000;
+  order: 2;
+}
+
+.color-3 {
+  background-color: #9a0000;
+  order: 3;
+}
+
+.color-4 {
+  background-color: #600;
+  order: 6;
+}
+
+.color-5 {
+  background-color: #300;
+  order: 5;
+}
+
+.color-6 {
+  order: 4;
+  background-color: #000;
+}
+
+@media (max-width: 899px) {
+  .color-3 {
+    background-color: #9a0000;
+    order: 4;
+  }
```

```suggestion
  .color-3 {
    order: 4;
  }
```
you don't change the color of the block, on the narrow screen 😁

---

Fixed!